tests/python_package_test/test_sklearn.py

```diff
@@ -1124,11 +1124,6 @@ def fit_and_check(eval_set_names, metric_names, assumed_iteration, first_metric_
     iter_min = min([iter_min_l1, iter_min_l2])
     iter_min_valid1 = min([iter_valid1_l1, iter_valid1_l2])
 
-    # training data as eval_set
```

Comment:
I think we shouldn't test this anymore but I'm open to modify it by adding an extra validation set so that we can check the training results